PWGMM/Tasks/uecharged.cxx

```diff
@@ -0,0 +1,336 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+/// \author Antonio Ortiz (antonio.ortiz.velasquez@cern.ch)
+/// \since November 2021
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Common/DataModel/Multiplicity.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/TrackSelectionDefaults.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Framework/HistogramRegistry.h"
+#include <TH1F.h>
+#include <TH2F.h>
+#include <TF1.h>
+#include <TRandom.h>
+#include <cmath>
+
+// TODO: implement flags for all, phys. sel., good vertex, ...
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct uecharged {
```

Comment:
To make sure that task would get a name `ue-charged` in the processing, you can use capitalization in the struct name: `UeCharged`